internal/invocation/invocation_service.go

```diff
@@ -68,15 +69,17 @@ func NewService(
 		groupLostCh:     make(chan *GroupLostEvent),
 		invocations:     map[int64]Invocation{},
 		handler:         handler,
-		eventDispatcher: eventDispacher,
+		eventDispatcher: eventDispatcher,
 		logger:          logger,
 		state:           ready,
+		executor:        newStripeExecutor(5, 100),
```

Comment:
How do you decide the size and count of the queues ?

Reply:
Thanks for reminding that!

I initially just used default value on Java Client for count, made something up for size. I will check if these are configureable with config file, and make necessary implementation to provide that.
If you also would like to see:
https://github.com/hazelcast/hazelcast/blob/fde1a42f93233858f206f660b048abfa3a7ac51a/hazelcast/src/main/java/com/hazelcast/client/properties/ClientProperty.java#L63

Reply:
Btw, default size looks 1m on java side.

Reply:
Yes you are right! I followed that implementation with https://github.com/hazelcast/hazelcast-go-client/pull/696/commits/c4dc047d11121599b217dcaf4ecb87967d3fb9c9

Reply:
I didn't understand, Will you keep it same with java or leave it 100? It looks **Outdated** but I couldn't see any change.

Reply:
@yuce also commented on this after I implemented your suggestions. For now leaving it only default values will be simpler to manage, considering that it already scales it self with the number of the CPU's dynamically.
Codebase is ready for implementing the configuration bits with ease. We thought, let's consider this in the feature if there is some issues to configure manually.
Does that sound good to you too?